script/Ciel/Day 14/14_01B_03_CIEL14B_6.txt

```diff
@@ -30,6 +30,7 @@ Exhaustion hits me all at once,
 [sha:6ef8ac365884b16305307171dce6552afcd0b6d1]{
 -- Page 286, Offset 41089.
 -- 世界は元通りになっても、世界が仕舞われている時にこちらに取り残されていた俺の体は『無かった事』にはならない。
+// Maybe change rewound / 無かった事
```

Comment:
reverted?

Reply:
I think this comment was left in case the current (which is the updated form from the TLC review) could still do with being made more clear. Guess it's fine if Petrikow didn't say anything?

Reply:
Nvm there's a comment updating this one later